tests/pytorch/test_onnx_export.py

```diff
@@ -959,7 +959,7 @@ def test_export_multihead_attention(
     input_ln_str = "_input-ln" if input_layernorm else ""
     fname = f"te.multihead_attention{fp8_str}{attn_mask_str}{attn_type_str}{input_ln_str}{fuse_qkv_str}{dtype_str}.onnx"
 
-    model = te.transformer.MultiHeadAttention(
+    model = te.attention.MultiHeadAttention(
```

Comment:
This is a user-visible change though, right?

Reply:
If they are using `MultiHeadAttention` then yes, but that's not a part of the API.

Reply:
The usage of the official API remains the same from user perspective, but any other internal function or module we can't guarantee.